content/articles/image-watermarking-with-opencv-python/index.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: engineering-education
+status: draft
+published: false
+url: /image-watermarking-with-opencv-python/
+title: Multiple Image Watermarking Using OpenCV with Python
+description: The objective of this tutorial is to help the reader understand the concept of how one can automate the process of image watermarking on multiple images with less effort and time.
+author: atonya-dennis
+date: 2021-09-22T00:00:00-10:16
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/image-watermarking-with-opencv-python/hero.jpg
+    alt: OpenCV Example image
+---
+
+Watermarks play an important part in businesses and online content credit making. They provide the author of digital artifacts rights or credits, preventing the content from being stolen or duplicated without the owner's permission. They could be as a distinctive and one-of-a-kind logo, signature, or stamp. Making these watermarks using Photoshop or manually requires more effort and time. We made this easier than ever using OpenCV with Python to automate watermarking multiple images.     
+<!--more-->
+We will cover as much as possible in this article to make the reader understand OpenCV, a python library used in the automation of image watermarking and in many other real-life scenarios.
+
+### Table of contents
+
+- [Prerequisites](#prerequisites).
+- [OpenCV Library](#opencv-library).
+- [Image Watermarking](#image-watermarking).
+  - [Loading Logo Image](#loading-logo-image).
+  - [Getting Images to Be Watermarked](#getting-images-to-be-watermarked).
+  - [Placing Watermark](#placing-watermark).
+- [Saving the Watermarked Images](#saving-the-watermarked-images).
+- [Conclusion](#conclusion).
+
+### Prerequisites
+To follow through this tutorial, the reader should:
+- Have a prior understanding of the basic python and OpenCV library concepts.
+- Know how to read, write and show images using OpenCV.
+- Have knowledge on how to use Pycharm IDE.
+
+Use this [page](https://www.jetbrains.com/pycharm/download/#section=windows) to download Pycharm IDE and install it on your machine.
+
+### OpenCV Library
+[OpenCV](https://www.mygreatlearning.com/blog/opencv-tutorial-in-python/) is a handy python library that is open source meant for computer vision processing functionalities as well as doing other critical operations such as tracking camera movements, extracting 3D models of any object, creating image filters and watermarks. It is a great tool for the automation of image watermarks as we will see in this tutorial.
```

Comment:
This explanation of OpenCV is too brief. Hold the hand of your reader. This brief intro is not enough to understand the library. You've mentioned that you'll cover as much as possible in this article to make the reader understand OpenCV. Please explain in detail.